typescript/hardhat/src/TestAbacusDeploy.ts

```diff
@@ -164,6 +172,11 @@ export class TestAbacusDeploy extends TestDeploy<
     const latestIndex = await outbox.count();
     if (latestIndex.eq(checkpointedIndex)) return responses;
```

Comment:
any idea what's happening in this check here? Not sure why it's useful but maybe we need to change this now that checkpointedIndex is not 1 indexed?

Reply:
Yes this is basically the same criteria as the checkpoint submitter. I.e. dont checkpoint if there are no more new messages